<?xml version="1.0" encoding="utf-8"?>
<launch>
    <node name="rviz" pkg="rviz" type="rviz" output="screen"
           args="-d $(find nav_traj_plan)/config/traj.rviz"> 
    </node>

    <node pkg="nav_traj_plan" type="traj_plan_3D" name="traj_plan_3D" output="screen">
        <!-- node parameters -->
        <rosparam file="$(find nav_traj_plan)/config/example0_params.yaml" command="load" />
    </node>

    <include file="$(find nav_traj_plan)/launch/map_server.launch"/>

</launch>